packages/react-vapor/src/components/calendar/Calendar.tsx

```diff
@@ -280,13 +280,16 @@ export class Calendar extends React.Component<ICalendarProps, any> {
         );
 
         if (this.props.countdown) {
-            const countdownDays = moment()
-                .date(moment().month(this.props.months[selectedMonth]).daysInMonth())
-                .fromNow(true);
+            const daysInMonth = moment().month(this.props.months[selectedMonth]).daysInMonth();
+            const daysLeftInMonth = moment().date(daysInMonth).fromNow(true);
+            const currentDayOfMonth = moment().date();
+
+            // Datepicker uses below logic to avoid displaying 'A few [hours|minutes|seconds] left'
+            const countdownTitle = currentDayOfMonth === daysInMonth ? 'Last day' : `${daysLeftInMonth} left`;
```

Comment:
I haven't tested it but could we go with something like that? I find it easier to read
```suggestion
            const endOfMonth = moment().endOf("month");
            const daysLeftInMonth = endOfMonth.diff(moment(), 'days');

            // Datepicker uses below logic to avoid displaying '0 days left'
            const countdownTitle = daysLeftInMonth === 0 ? 'Last day' : `${daysLeftInMonth} days left`;
```